### PR TITLE
Ch844 import queries from explore

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      subscription {
+        observationEdit(programId:"p-2") {
+          id
+        }
+      }
+    """
+  }
+
+  @GraphQL
+  trait ProgramCreateObservation extends GraphQLOperation[ObservationDB] {
```
Comment (cquiroz) [inline modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala:64]: Do you need to create/delete observations in `observe`?

File: project/Settings.scala [hunk 2]
```diff
@@ -32,7 +32,7 @@ object Settings {
     val scalaXml            = "1.2.0"
     val catsTime            = "0.3.4"
 
-    val http4sVersion  = "0.23.0-M1"
+    val http4sVersion  = "1.0.0-M21"
```
Comment (cquiroz) [inline project/Settings.scala:35]: Brave

File: project/plugins.sbt [hunk 3]
```diff
@@ -34,4 +34,4 @@ addSbtPlugin("org.scala-sbt.plugins" % "sbt-xjc" % "0.10")
 
 
 
-onLoad in Global := { s => "dependencyUpdates" :: s }
+Global / onLoad := { s => "dependencyUpdates" :: s }
```
Comment (cquiroz) [inline project/plugins.sbt:37]: do we still need this in the age of `scala-steward`?

**Review by cquiroz — APPROVED** [review]
Looks good
It seems you got the queries from explore. If so, should we share that code into a lib? Or is it going to evolve independently?

File: modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala (hunk 1 above)
Comment (jluhrs) [reply modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala:64]: No

File: project/Settings.scala (hunk 2 above)
Comment (jluhrs) [reply project/Settings.scala:35]: Clue made me do it

**Comment by jluhrs** [comment]
I was talking about the same with Raúl. There some pieces that could be shared, specially the decoders.

> Looks good
> It seems you got the queries from explore. If so, should we share that code into a lib? Or is it going to evolve independently?

**Pull request merged**